hack/undeploy-karmada.sh

```diff
@@ -44,6 +44,13 @@ ETCD_HOST_IP=$(kubectl get pod -l app=etcd -n karmada-system -o jsonpath='{.item
 # clear all in namespace karmada-system
 kubectl delete ns karmada-system --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
 
+# clear global resources
```

Comment:
I guess this comment can be removed.

Reply:
aggreed 
removing it looks better
done